Rename the world from DK to Workstation

tas50 opened the pull request:
There is one tool and it's called Workstation. Let's pretend we never called it DK.

I'll fully admit this is "rage coding" at it's best here. We're actively pushing users to use Worksktatioin, but our docs still point them to DK. There will be no DK next year and we're setting people up for a bad time. This needs to get fixed ASAP,

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Rename the world from DK to Worksktation

There is one tool and it's called Workstation. Let's pretend we never called it DK.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix links and move everything under the Chef Workstation nav group

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Collapse the workstation windows page into the overall workstation page

We need to stop partially duplicating platform specific docs throughout the site

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix extra `s

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove references to EOL Windows releases

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix another header

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
dtag fix

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove the DK warning and who cares about omnibus

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
No one cares we build from source

Signed-off-by: Tim Smith <tsmith@chef.io>

kagarmoe commented:
I'm not opposed to this, but we need to make sure that we document the unique Workstation content in chef-web-docs.

tas50 committed:
More updates to get a working build

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix a dtag

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
workstation -> Chef Workstationn

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Update the platform overview

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove old DK blog post

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
We support EL 8 in our package repos

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Require SLES 12 SP4+ per our policy

SP3 went EOL in june

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Rework how we describe workstation

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Lets not mention manage anymore

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Nuke another place we were mentioning manage

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Focus on the workstation tools people care about

We're trying to move away from Berkshelf so don't mention that. No one cares about Fauxhai since it just silently works. They do care about Test Kitchen drivers though.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Chef -> Chef Infra and remove old workstation links

Match branding a bit more

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove the main page link to the management console

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Common Functionality -> Common Resource Functionality

That's what this actually is

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Kill windows_dk in favor of the install_workstation page

There's already a redirect for this.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Update our openssl example output

We're very good about keeping this file up to date. Let's not make it seem like we're 5 years out of date.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Move duplicate workstation tools in the nav bar

Let's put all the tools just under workstation

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Cleanup redirects and the navbar a bit more

Let's make sure things are only in a single place

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Move Infra tools out of the Workstation tools list

Move these into the Infra features section

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove the text editor section from workstation install

This is in workstation setup where it belongs

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Don't link to a doc that has no helpful information

There's no steps for Windows users here

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix formatting issues

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove reference to community tools

The whole thing is one giant community tool. It seems silly to group them that way.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Move the note about DK to the About Workstation page

You should know this before you get to the installation

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Use the same list of components for Workstation everywhere

Focus on the tools and not on things like Ruby that don't matter

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Minor wording cleanup

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Clarify that we're talking about Chef Server 12+

When we wrote this 13 wasn't out.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Update chef gem example

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
More DK -> Workstation updates

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Remove knife_spork page

This isn't in Workstation and it's not something we've promoted for a while.

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Move DK -> Workstation

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Fix link that's failing the build

Signed-off-by: Tim Smith <tsmith@chef.io>

Commit:
Rework left nav

Signed-off-by: IanMadd <imaddaus@chef.io>

mjingle commented on chef_master/source/config_yml_kitchen.rst:
```diff
@@ -133,7 +133,7 @@ Kitchen can configure the chef-zero provisioner with the following Chef-specific
      - **This will be deprecated in a future version.**
    * - ``chef_omnibus_install_options``
-     - Use to specify the package to be installed. Possible values: ``-P chef`` (for Chef Infra Client) and ``-P chefdk`` (for the Chef Infra Client that is packaged as part of ChefDK). Use ``-n`` to specify the nightly build. For example: ``-P chefdk`` or ``-n -P chefdk``. **This will be deprecated in a future version.** See the ``product_name``, ``product_version``, and ``channel`` settings ins ...
+     - Use to specify the package to be installed. Possible values: ``-P chef`` (for Chef Infra Client) and ``-P chef-workstation`` (for the Chef Infra Client that is packaged as part of Chef Workstation). Use ``-n`` to specify the nightly build. For example: ``-P chef-workstation`` or ``-n -P chef-workstation``. **This will be deprecated in a future version.** See the ``product_name``, ``product ...
```
Is the bolded deprecation note still applicable with this change from DK -> workstation?

tas50 replied:
Should still be valid

mjingle commented on chef_master/source/fips.rst:
```diff
@@ -26,7 +26,7 @@ Supported Products
 
 * `Chef Infra Client </fips.html#how-to-enable-fips-mode-for-the-chef-client>`__
-* `ChefDK </fips.html#how-to-enable-fips-mode-for-workstations>`__
+* `Chef Workstation <fips.html#how-to-enable-fips-mode-for-workstations>`__
```
```suggestion
* `Chef Workstation </fips.html#how-to-enable-fips-mode-for-workstations>`__
```

mjingle commented on chef_master/source/glossary.rst:
```diff
@@ -57,8 +57,8 @@ Glossary
-**kitchen**
-   Kitchen is an integration framework that is used to automatically test cookbook data across any combination of platforms and test suites. Kitchen is packaged in ChefDK.
+**Test Kitchen**
+   Test itchen is an integration framework that is used to automatically test cookbook data across any combination of platforms and test suites. Test Kitchen is packaged in Chef Workstation.
```
```suggestion
   Test Kitchen is an integration framework that is used to automatically test cookbook data across any combination of platforms and test suites. Test Kitchen is packaged in Chef Workstation.
```

mjingle commented on chef_master/source/install_workstation.rst:
```diff
@@ -0,0 +1,143 @@
+=====================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/install_workstation.rst>`__
+
+Start your infrastructure automation quickly and easily with `Chef Workstation <https://www.chef.sh/>`__ Chef Workstation gives you everything you need to get started with Chef — ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software — all in one easy-to-install package.
```
```suggestion
Start your infrastructure automation quickly and easily with `Chef Workstation <https://www.chef.sh/>`__ . Chef Workstation gives you everything you need to get started with Chef — ad hoc remote execution, remote scanning, configuration tasks, cookbook creation tools as well as robust dependency and testing software — all in one easy-to-install package.
```

tas50 committed:
Update chef_master/source/fips.rst

Signed-off-by: Tim Smith <tsmith@chef.io>

Co-Authored-By: mjingle <5807053+mjingle@users.noreply.github.com>

tas50 committed:
Update chef_master/source/install_workstation.rst

Signed-off-by: Tim Smith <tsmith@chef.io>

Co-Authored-By: mjingle <5807053+mjingle@users.noreply.github.com>

tas50 committed:
Update chef_master/source/glossary.rst

Signed-off-by: Tim Smith <tsmith@chef.io>

Co-Authored-By: mjingle <5807053+mjingle@users.noreply.github.com>

tas50 committed:
Chef -> Chef Infra

Signed-off-by: Tim Smith <tsmith@chef.io>

tas50 committed:
Update the overview graphic to include Workstation

Signed-off-by: Tim Smith <tsmith@chef.io>

Commit:
workstation-Chef Workstation edits

Signed-off-by: Mary Jinglewski <mjinglewski@chef.io>

mjingle reviewed (approved):
![tenor-192699211](https://user-images.githubusercontent.com/5807053/73775074-31b49f80-4753-11ea-9278-8282fc632e9c.gif)
